app/src/main/java/ar/com/wolox/android/example/ui/login/LoginActivity.java

```diff
@@ -0,0 +1,19 @@
+package ar.com.wolox.android.example.ui.login;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.wolmo.core.activity.WolmoActivity;
+
+/**
+ * Class Login Fragment
+ */
+public class LoginActivity extends WolmoActivity {
+    @Override
+    protected void init() {
+        replaceFragment(R.id.vActivityBaseContent, new LoginFragment());
```

Comment:
This is great. Read about the fragment newInstance convention and the start activity, to make this Perfect. If you don't find it, we can talk about it. It is just a good practice, it is not a must